### DISCUSSION TITLE
Running test suite against Oracle Database

### DISCUSSION THREAD

**Discussion by Kajalagg98** [sqlalchemy/sqlalchemy#7250] [Usage Questions] [answered]
Any db configurations required or any script required prior to running sqlalchemy test suite against oracle db?

**Comment by CaselIT** [comment, marked as answer]
Hi,

You should find all the information here https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst

**Reply by Kajalagg98** [reply]
Thanks!
